tenant-file/portal-app/src/component/export-toolbar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import StyledExportToolbar from './styles/ExportToolbarStyles';
+
+type Props = {
+  isExportToolsOpen: Boolean;
+};
+
+const ExportToolbar = ({ isExportToolsOpen }: Props) => {
+  return (
+    <StyledExportToolbar className={isExportToolsOpen ? '' : 'hide'}>
+      <span tabIndex={0}>
```

Comment:
Can you actually make these buttons with an onClick handler? All it has to do is console.log("something") for right now, but I just want to have the hooks in place for when we need them.